packages/dcos-checks/extra/dcos-checks.timer

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=DC/OS Checks Timer: timer for DC/OS Checks service
+[Timer]
+Unit=dcos-checks.service
+OnBootSec=15min
+OnUnitActiveSec=2m
```

Comment:
I think we want `OnCalendar` here. We want to activate this timer on a regular schedule, regardless of when/whether the corresponding unit was last activated or how long it took to run.

Is there a reason we can't set the interval lower? 60 seconds seems reasonable. Or maybe even lower, node checks aren't a lot of work.